### DISCUSSION TITLE
.net 4.5  我有多个数据库要访问，都是Oracle,我是要创建多个实例，还是用单个实例，在访问时切换链接字符串？

### DISCUSSION THREAD

**Discussion by netty2019** [dotnetcore/FreeSql#1373] [General]
.net 4.5  我有多个数据库要访问，都是Oracle,我是要创建多个实例，还是用单个实例，在访问时切换链接字符串？

**Comment by 2881099** [comment]
如果是同服务器使用 select * from db.table 能访问，就定义一个 IFreeSql。

如果不能，就用 FreeSqlCloud 分别 Use 访问。

**Reply by netty2019** [reply]
好的，谢谢
